contents/blog/why-we-made-surveys.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2023-10-19
+title: Why we've launched PostHog user surveys
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - joe-martin
+featuredImage: ../images/blog/posthog-survey.png
+featuredImageType: full
+category: Inside PostHog
+---
+
+Today, we’ve announced user surveys are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
```

Comment:
```suggestion
Today, we’re announcing [user surveys](LINK TO PRODUCT PAGE WHEN LIVE) are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
```

Reply:
Pricing link should link to Survey pricing page when it's live

Reply:
```suggestion
Today, we’ve announced [user surveys](/surveys) are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
```